<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Report</class>
 <widget class="QDialog" name="Report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="durationmin_d">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="durationmin_u">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>280</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="durationsec_d">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="durationsec_u">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>280</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5_separatoredurata">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>280</y>
     <width>31</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cadencefinal">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_durata">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Durata totale</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cadenza">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cadenza finale</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_titolo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>511</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>48</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Report finale</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_HR">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>HR finale</string>
   </property>
  </widget>
  <widget class="QLabel" name="HRfinal">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>460</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>570</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CHIUDI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>HOME</string>
   </property>
  </widget>
  <widget class="QLabel" name="durationsec_d_sat">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>340</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="durationmin_d_sat">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="durationmin_u_sat">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5_separatoredurata_sat">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>31</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_saturazione">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Saturazione</string>
   </property>
  </widget>
  <widget class="QLabel" name="durationsec_u_sat">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>340</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
